conda.recipe/meta.yaml

```diff
@@ -36,6 +35,7 @@ requirements:
     - six >=1.15.0
     - tqdm >=4.56.0
     - urllib3 >=1.26.4
+    - anaconda-cli-base >0.2.3
```

Comment:
we can now pin `anaconda-cli-base >=0.3.0`

Reply:
:thumbsup: